### ISSUE TITLE
Please fix color contrast issues in the demo

### ISSUE THREAD

**Issue by mgifford** [Choices-js/Choices#924]
**Describe the bug**
The demo page should be check for color contrast and have accessibility issues fixed.

**To Reproduce**
If you go here joshuajohnson.co.uk/choices/

Run https://wave.webaim.org/ on it and you see a great many contrast errors because of the painful choice of colors. 

**Expected behavior**
There should be clear and readable. Should comply with https://webaim.org/articles/contrast/

**Screenshots**
![Image of WAVE report](https://user-images.githubusercontent.com/116832/104341828-3851a800-54c8-11eb-9358-13c684c4eb3f.png)

**Comment by Xon** [comment]
I've picked #005F75 to replace #00bcd4, which provides a good contrast against white according to; https://dequeuniversity.com/rules/axe/4.9/color-contrast.

Added to the list of things to be merged into the big v11.0.0 update I've been working on
